### ISSUE TITLE
Consider adding alternative text for crown and crest

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-frontend#5807]
## What

Think about alternative text for the crown in the header and the footer and the crest in the footer.
We currently don't have any alternative text to describe or even mention the crown or the crest. 

The current alternative texts (not necessarily value for `alt` parameter) are:

* The crown in the header includes the word mark. It is linked and leads to the homepage. It's an`<svg>` with an `aria-label` of "GOV.UK".
* The crown in the footer does not include the word mark. It is not linked.  It's an`<svg>` with a `role="presentation". (The crown is not in the footer yet, but it will be soon.)
* The crest in the footer does not include any text (within the image itself, unless you count the motto, which is really too small to see). It is the background image of a link that goes to a National Archives page about the Crown copyright. And that link includes the text "© Crown copyright".

To summarise the alternative texts for those images...

| Image (text approximation) | Alternative text | Is in a link? |
| --- | --- | --- |
| 👑 GOV.UK | GOV.UK | yes |
| 👑 |  | no |
| 🛡️ © Crown copyright | © Crown copyright | yes |

## Why

We never describe or reference the images. We should explore if doing that would be valuable to people who cannot see the images (or cannot see them well enough).

We should find out what these images do for people who can see them. And if they do anything important, we should consider those images to be meaningful and add (to) alternative text to describe them.

My (untested) assumptions:
Visually both images give the page a more official look.
They help establishing trust.
They reference something a user might want to be looking for.
And they signal the start and the end of the page.

If that is correct, they should have alternative text that conveys those things.

It's worth noting that this has never been flagged in a WCAG audit. Therefore it might not be a problem and is not an urgent thing to fix.

## Who needs to work on this

Content designer, designer, accessibility specialist, user researcher

## Who needs to review this

The team, and maybe someone from GOV.UK should be involved as well

## Done when

- [ ] Have evidence for meaning of those images to users (either through our own research or someone else's)
- [ ] Decided if those images are meaningful to our users
- [ ] If they are, write alternative text for them and implement it

**Comment by owenatgov** [comment]
This was flagged by Digital Identity in an accessibility question:

>The small crown logo in the left top of the footer isn’t described to screenreader users. While the image might be considered decorative, I think it’s worth documenting why this decision has been made and why the image is needed (e.g. to increase trust) – this may mean we need to add alt text to the image or otherwise find ways of communicating the information it’s providing for assistive technologies.

**Comment by 36degrees** [comment]
The lack of alt text on the crown came up again in #7307.
